process/due-diligence-guidelines.md

```diff
@@ -0,0 +1,142 @@
+  decide for yourself whether or not you have enough info to make a strong, informed call on that item.
+  * If so, write it down, with motivation.
+  * If not, jot down what information you feel you're missing.
+  * Also take note of what unanswered questions the community might have posted in the PR review that you consider
+    to be critically important.
+    
+### Some example questions that will ideally need clear answers
+
+Most of these should be covered in the project proposal document.  The
+due diligence exercise involves validating any claims made there,
+verifying adequate coverage of the topics, and possibly summarizing
+the detail where necessary.
+
+#### Technical
+
+* An architectural, design and feature overview should be available.
+  ([example](https://github.com/docker/notary/blob/master/docs/service_architecture.md),
+  [example](https://github.com/docker/notary/blob/master/docs/command_reference.md))
+* What are the primary target cloud-native use cases?  Which of those:
+  * Can be accomplished now.
+  * Can be accomplished with reasonable additional effort (and are ideally already on the project roadmap).
+  * Are beyond the current roadmap.
+* What are the current performance, scalability and resource consumption bounds of the software?  Have these been explicitly tested?
+  Are they appropriate given the intended usage (e.g. agent-per-node or agent-per-container need to be lightweight, etc)? 
+* What exactly are the failure modes?  Are they well understood?  Have they been tested?  Do they form part of continuous integration testing?
+  Are they appropriate given the intended usage (e.g. cluster-wide shared services need to fail gracefully etc)?
+* What trade-offs have been made regarding performance, scalability, complexity, reliability, security etc?  Are these trade-offs explicit or implicit?
+  Why?  Are they appropriate given the intended usage?  Are they user-tunable? 
+* Code quality.  Does it look good, bad or mediocre to you (based on a spot review).  How thorough are the code reviews? Substance over form.
+  Are there explicit coding guidelines for the project?
+* Dependencies.  What external dependencies exist, do they seem justified, and are their licenses compatible with their usage and CNCF policies?
+  CNCF staff will handle the full legal due diligence.
+* What is the release model?  Versioning scheme?  Evidence of stability or otherwise of past stable released versions?
+* What is the CI/CD status?  Do explicit code coverage metrics exist? If not, what is the subjective adequacy of automated testing?
+  Do different levels of tests exist (e.g. unit, integration, interface, end-to-end), or is there only partial coverage in this regard?  Why?
+* What licensing restrictions apply?  Again, CNCF staff will handle the full legal due diligence.
+* What are the recommended operational models?  Specifically, how is it operated in a cloud-native environment, such as on Kubernetes?
+
+#### Team
```

Comment:
I would call this subsection "Project" and rename the other subsection below.

Reply:
Agreed, done.